Structure_Engine/Query/Geometry.cs

```diff
@@ -84,9 +84,11 @@ public static IGeometry Geometry(this RigidLink link)
                 {
                     if (sn?.Position != null)
                         lines.Add(new Line() { Start = link.PrimaryNode.Position, End = sn.Position });
-                }   
+                }
             }
-            
+            else
+                return null;
+
```

Comment:
Not sure what benefit this addition has? Retuning empty geometry and protected from crashing as is anyway?